### PR TITLE
Getting the voice recognition working

### PR DESCRIPTION
**Pull request by bahaa96**
The current code is working but need some optimizations and you may find bugs 😺 

you might find an issue in the voice recognition like not getting the right word I'm currently investigating this to see is it something about the size of the stream's chunks or the language itself because this error is rare in recognizing English speech.

### CODE REVIEW

**Comment by tifa2UP** [comment]
Does the voice server freeze? Still takes a bit of time before getting the initial response

**Comment by tifa2UP** [comment]
lgtm! a few minor comments about styles. Feel free to merge once you fix them.

@bahaa96 can you add the link to the backend voice server PR here as well?

**Comment by bahaa96** [comment]
> lgtm! a few minor comments about styles. Feel free to merge once you fix them.
> 
> @bahaa96 can you add the link to the backend voice server PR here as well?

Where are these comments ? 

here's the backend code I'll add it to the org. sooner.
https://github.com/bahaa96/Tarteel-voice

**Comment by tifa2UP** [comment]
> > lgtm! a few minor comments about styles. Feel free to merge once you fix them.
> > @bahaa96 can you add the link to the backend voice server PR here as well?
> 
> Where are these comments ?
> 
> here's the backend code I'll add it to the org. sooner.
> [bahaa96/Tarteel-voice](https://github.com/bahaa96/Tarteel-voice)

NVM, the comments were pending. You should be able to see them now

File: src/pages/Recognition/index.tsx [hunk 1]
```diff
@@ -66,166 +62,96 @@ class Recognition extends React.Component<IProps, IState> {
...
-      };
-    });
-  };
-  startRecognition = () => {
     this.setState({
-      isRecording: true,
+      partialQuery: interimTranscript,
     });
-    this.recognition = new webkitSpeechRecognition();
-    this.recognition.lang = 'ar-AE';
```
[the remark this answers is not in the document]
Comment (bahaa96) [reply src/pages/Recognition/index.tsx]: yes

File: src/components/Headline.tsx [hunk 2]
```diff
@@ -33,14 +51,17 @@ const HeadlineContainer = styled.div`
   max-width: ${props => props.width}%;
   h1 {
     span {
-      font-size: ${props => props.headtagSize ? props.headtagSize + 'px' : 'inherit'};
```
[the remark this answers is not in the document]
Comment (moumen-soliman) [reply src/components/Headline.tsx]: no, why?

[the remark this answers is not in the document]
Comment (tifa2UP) [reply src/components/Headline.tsx]: It's producing different formats for the same line

[the remark this answers is not in the document]
Comment (moumen-soliman) [reply src/components/Headline.tsx]: I think my prettier update in the last PR not commit, @bahaa96 help me with that and do it 👍

[the remark this answers is not in the document]
Comment (bahaa96) [reply src/components/Headline.tsx]: IDK which is better ? I didn't make this manually I just ran the script "format:src"

**Pull request merged**